packages/core/src/ui/GeoQuery/GeoQuery.tsx

```
@@ -0,0 +1,810 @@
+      map.on("styledata", () => {
+        if (lastFcRef.current !== null || map?.getSource("portaljs-geoquery")) {
+          paintOverlay(lastFcRef.current);
+        }
+      });
+
+      // --- DuckDB-Wasm + spatial ---
+      let workerUrl: string | null = null;
+      try {
+        const bundles = duckdbMod.getJsDelivrBundles();
+        const bundle = await duckdbMod.selectBundle(bundles);
+        workerUrl = URL.createObjectURL(
```

ESLint

[error] 'URL' is not defined.

(no-undef)